### DISCUSSION TITLE
[A-L] (26 Gennaio 2024) Esercizio 3

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#401] [Esami A-L]
Una relazione binaria $\Large R$ su un insieme $\Large A$ è detta simpatica se ha la seguente proprietà:

<div align="center">

per ogni $\Large a, b, c, d \in A$, se $\Large aRb$ e $\Large bRc$ e $\Large cRd$ allora $\Large aRd$.

</div>

Indicare se i seguentipunto sono veri o falsi.

### Fila A

- 1\. Ogni relazione transitiva è simpatica. [1 punto]

- 3\. Nessuna relazione simpatica è transitiva. [1 punto]

- 4\. La seguente relazione binaria  $\Large R$ sull’insieme  $\Large \\{1,\ 2,\ 3,\ 4,\ 5\\}$ è simpatica? [1 punto]

$$
    \Large R = \\{(1, 2),\ (1, 4),\ (1, 5),\ (2, 4),\ (2, 2),\ (3, 5),\ (3, 1),\ (4, 5),\ (5, 1)\\}
$$

### Fila B

- 1\. Qualche relazione transitiva è simpatica. [1 punto]

- 3\. Nessuna relazione transitiva non è simpatica. [1 punto]

- 4\. La seguente relazione binaria R sull’insieme {1, 2, 3, 4, 5} è simpatica? [1 punto]

$$
    \Large R = \\{(1, 1),\ (1, 2),\ (1, 4),\ (1, 5),\ (2, 4),\ (5, 4),\ (5, 1),\ (5, 2),\ (5, 5)\\}
$$

### Entrambe le file

- 2\. Ogni relazione simpatica è transitiva. [1 punto]

- 5\. Scrivere una relazione simpatica sull’insieme dei numeri naturali $\Large \mathbb{N}$. [1 punto]

**Comment by cek753** [comment]
A: V F F
B: V V V
A-B: Falso, R={(1,1),(1,2),(2,3),(1,3)}
